components/Navigation.tsx

```diff
@@ -27,174 +25,303 @@ type subroutes = subroute[];
     subroutes: [
       {
         type: 'route',
-        title: 'Command line tool',
-        path: '/commands',
+        title: 'Avo 101 for developers',
+        path: '/implementation/devs-101',
+      },
+      {
+        type: 'group',
+        group: 'Getting started guides',
       },
       {
         type: 'route',
-        title: 'Programming languages',
-        path: '/languages',
+        title: 'Getting started with Avo Functions',
+        path: '/implementation/start-using-avo-functions',
       },
       {
         type: 'route',
-        title: 'Analytics tools',
-        path: '/analytics',
+        title: 'Getting started to receive data with a Custom Destination',
+        path: '/implementation/start-custom-destination',
       },
       {
         type: 'route',
-        title: 'Custom destinations',
-        path: '/custom-destinations',
+        title: 'Getting started with the Avo Visual Debuggers',
+        path: '/implementation/start-using-visual-debugger',
       },
       {
         type: 'route',
-        title: 'Regression checking',
-        path: '/regression',
+        title: 'Getting started to use standalone Inspector',
+        path: '/implementation/start-using-inspector',
       },
       {
         type: 'route',
-        title: 'Data validation',
-        path: '/data-validation',
+        title: 'Getting started to use Inspector alongside Avo Functions',
+        path: '/implementation/start-using-inspector-with-avo-functions',
       },
       {
         type: 'route',
-        title: 'Mobile debuggers',
-        path: '/mobile-debuggers',
+        title:
+          'Getting started to implement tracking plan changes in code with Avo',
+        path: '/implementation/start-implementing-tracking-changes',
+      },
+      {
+        type: 'group',
+        group: 'Tools',
+      },
+      {
+        type: 'route',
+        title: 'Supported programming languages',
+        path: '/implementation/supported-programming-languages',
+      },
+      {
+        type: 'route',
+        title: 'Command line tool',
+        path: '/implementation/cli',
+      },
+      {
+        type: 'route',
+        title: 'Logs',
+        path: '/implementation/logs',
+      },
+      {
+        type: 'group',
+        group: 'Advanced guides and best practices',
       },
-    ],
-  },
-
-  {
-    title: 'Inspector',
-    path: '/inspector',
-    iconName: 'heartbeat',
-    subroutes: [
       {
         type: 'route',
-        title: 'Inspector SDK',
-        path: '/inspector/sdk',
+        title: 'Avo and git',
+        path: '/implementation/avo-and-git',
+      },
+      {
+        type: 'route',
+        title: 'Avo and unit tests',
+        path: '/implementation/avo-and-unit-tests',
+      },
+      {
+        type: 'route',
+        title: 'Avo in the CI',
+        path: '/implementation/avo-in-the-ci',
+      },
+      {
+        type: 'route',
+        title: 'Subscribe to tracking plan changes webhook',
```

Comment:
Wonder if we can get away with shorter version in the navbar:
```suggestion
        title: 'Subscribe to changes with webhooks',
```